tests/unit-js/src/test/scala/tests/js/JsSuite.scala

```diff
@@ -247,7 +292,27 @@ class JsSuite extends BaseMarkdownSuite {
       baseSettings.copy(
         site = baseSettings.site.updated("js-classpath", Classpath(noScalajsDom).syntax)
       )
-    }
+    },
+    compat = Map(
+      Compat.Scala3 ->
+        """
+          |error:
+          |no-dom.md:3 (mdoc generated code)
+          | value scalajs is not a member of org
+          |def run0(node: _root_.org.scalajs.dom.raw.HTMLElement): Unit = {
+          |
+          |112
```

Comment:
What are these two numbers? :thinking:

Reply:
I was going to say no idea, but then I figured it out :D https://github.com/scalameta/mdoc/pull/504/commits/aa068bf0d6c9dd288a58f04cb646124349f3f6d7#diff-805fd83a6416d9a09a57c82fd1b9b21b3d854e248a33254d26bda2b2334bee23L277